third_party/tailscale/distsign/roots.go

```diff
@@ -11,7 +11,7 @@ import (
 	"path/filepath"
 	"sync"
 
-	"github.com/leptonai/gpud/rootkeys"
+	"github.com/leptonai/gpud/third_party/tailscale/distsign/rootkeys"
```

Comment:
Actually we can doing as following to reuse the tailscale tests
```go
//go:embed keys
var rootsFS embed.FS
```

and move the keys folder to distsign...